src/firefly/js/visualize/ui/VisCtxToolbarView.jsx

```diff
@@ -656,3 +657,16 @@ MultiImageControllerView.propTypes= {
     plotView : PropTypes.object.isRequired,
 };
 
+
+function getHipsCubeDesc(plot) {
+    if (!isHiPS(plot)) return '';
+    const {hipsProperties}= plot;
+    const {data_cube_crpix3, data_cube_crval3, data_cube_cdelt3, data_cube_bunit3=''}= hipsProperties;
+    if (!data_cube_crpix3 || !data_cube_crval3 || !data_cube_cdelt3) return ''
+    const crpix3= Number(data_cube_crpix3);
+    const crval3= Number(data_cube_crval3);
+    const cdelt3= Number(data_cube_cdelt3);
+    if (isNaN(crpix3) || isNaN(crval3) || isNaN(cdelt3)) return '';
+    const value = crval3 + ( plot.cubeIdx - crpix3 ) * cdelt3;
+    return `${numeral(value).format('0.00000')} ${data_cube_bunit3}`
```

Comment:
for cube case like 'CALIFA DR2 V500...', 'data_cube_bunit3' is undefined, and the display shows nothing. 
for cube case like  'HI4PI HI Survey (individual cubes)', 'data_cube_bunit3' is defined as 'null', and the display shows 'null'.  Does 'null' mean some unit in default? or could the display also show nothing instead of 'null'?

Reply:
This is because `data_cube_bunit3` is the string "null".  I think we are doing the right think by the standard.  However, I think I will just look for the string 'null' and ignore it.